vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.h

```diff
@@ -110,6 +110,10 @@ class ReactInstanceWin final : public Mso::ActiveObject<IReactInstanceInternal,
   friend struct LoadedCallbackGuard;
   void OnReactInstanceLoaded(const Mso::ErrorCode &errorCode) noexcept;
 
+#if defined(USE_V8)
+  static std::string getApplicationLocalFolder();
```

Comment:
>) [](start = 47, length = 1)

noexcept?

Reply:
It looks like the underlying winrt API is not noexcept

---
In reply to: [387942112](https://github.com/microsoft/react-native-windows/pull/4234#discussion_r387942112) [](ancestors = 387942112)